sdk/src/main/scala/com/horizen/account/mempool/MempoolMap.scala

```diff
@@ -267,13 +300,20 @@ class MempoolMap(
     var destMap = newExecTxs
     var haveBecomeNonExecutable = false
     var maxNonceGapExceeded = false
+    var currAccountSlots = 0L
```

Comment:
do we need to use `Long` type here ?
same question about `txSizeInSlot` method.
`Int` must be more than enough.

Reply:
This is already solved in PR: https://github.com/HorizenOfficial/Sidechains-SDK/pull/728